core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/setoperators/ExceptOperator.kt

```diff
@@ -0,0 +1,13 @@
+package ch.ergon.dope.resolvable.clause.model.setoperators
+
+import ch.ergon.dope.resolvable.clause.ISelectOffsetClause
+
+class ExceptOperator(
```

Comment:
Do we need these three classes (_except_, _set_ and _union_)? Can't we just create always directly a _SetOperator_?

Reply:
as discussed I changed it to an enum, now it's only the set operator, so I consider this done